app/src/pages/inside/projectSettingsPageContainer/content/integrations/integrationsList/integrationInfo/integrationCollection/integrationCollectionItem/integrationCollectionItem.jsx

```diff
@@ -45,12 +45,17 @@ export const IntegrationCollectionItem = ({
   item,
 }) => {
   const [connected, setConnected] = useState(true);
-  const projectKey = useSelector(activeProjectKeySelector);
-  const payloadProjectKey = useSelector(urlProjectKeySelector);
+  const projectKey = useSelector(projectKeySelector);
+  const { organizationSlug, projectSlug } = useSelector(urlOrganizationAndProjectSelector);
```

Comment:
Mark with `TODO`.
The fallback here need to be replaced with more robust value.

Reply:
Done.